saq/worker.py

```diff
@@ -214,7 +214,7 @@ async def process(self):
             logger.error(error)
 
             if job:
-                if job.attempts >= job.retries:
```

Comment:
I think this was always wrong. I think this was implemented as if "retries" was actually "max_attempts".

Changing this corrects the semantics. But it's also a breaking change in behavior. Let me know if that's OK.

Reply:
breaking changes are always ok.

you're right, in my mind, retries means max_attempts.

and so job.attempts >= job.retries is correct because retries = 1 means that it will retry once
retries 2 means it will retry twice.

do you think this really needs to be changed?

Reply:
i think that we just have different interpretations of the word "retries" and so the current implementation is what i intended

Reply:
_re_try means trying _again_ haha.

But that's OK. If that's how you interpret it, I can change it back.